index.html

```diff
@@ -1,15 +1,233 @@
+    <p class="img--price"><i class="fas fa-rupee-sign"></i>  9999</p></div>
+
+
+
+    
+  <div class="card__newArrival p-0-2">
+    <img src="/assets/landing page Images/category watch 3 .png" alt="">
+    <p class="img--info">Rado ferrari finish </p>
+    <p class="img--price"><i class="fas fa-rupee-sign"></i>  9999</p></div>
+
+
+
+    
+  <div class="card__newArrival">
+    <img src="/assets/landing page Images/category watch 4 .png" alt="">
+    <p class="img--info">Hubolt sailor finish </p>
+    <p class="img--price"><i class="fas fa-rupee-sign"></i>  9999</p></div>
+</div>
+ 
+
+</div>
+
+
+<div class="section--content">
+
+  <p class="section--content__para">Time is Precious , And So are you . <br>
+    We have Selected the Best Watches from Whole World . <br>
+    To make Every Luxury Brand available under one website <br>
+
+  </p>
+<img src="//unsplash.it/1080/720?car" alt="">
+
+
+</div>
+
+
+<div class="section ">
+
+  <h2>Popular Collections </h2>
+<div class="popular">
+<div class="collection--item">
+<div class="ladies">
+   <img src="/assets/landing page Images/vitae-london-331tAGMqa8Y-unsplash.jpg" alt="">
+  
+  <div class="img--info">Ladies</div>
+  <div class="textAlign--center"><a href="">Discover</a></div>
+  </div>
+</div>
+<div class="collection--item">
+  <div class="ladies">
+     <img src="/assets/landing page Images/klim-musalimov-nmhZgiSU8_0-unsplash.jpg" alt="">
+    
+    <div class="img--info">Smart Watches </div>
+    <div class="textAlign--center"><a href="">Discover</a></div>
+    </div>
+  </div>
+
+  <div class="collection--item">
+    <div class="ladies">
+       <img src="/assets/landing page Images/men section .jpg" alt="">
+      
+      <div class="img--info">Gents</div>
+      <div class="textAlign--center"><a href="">Discover</a></div>
+      </div>
+    </div>
+
+ </div>
+  
+
+</div>
+<footer>
+
+
+
+
+<div class="footer--section">
+<h2 class="footer--heading">Collections</h2>
+
+<ul>
+  <li>Rolex</li>
+  <li>Omega</li>
+  <li>Cartier</li>
+  <li>Montblanc</li>
+  <li>Hublot</li>
+  <li>SmartWatches</li>
+ 
+</ul>
+
+</div>
+
+
+<div class="footer--section">
+  <h2 class="footer--heading">Front Fist </h2>
+  
+  <ul>
+    <li>Career</li>
+    <li>Press</li>
+  
+    <li>News</li>
+    <li>Visit us </li>
+   
+  </ul>
+  
+  </div>
+
+  
+
+
+    <div class="footer--section">
+      <h2 class="footer--heading">Services</h2>
+      
+      <ul>
+        <li>Maintainence</li>
+        <li>Return Your Watch </li>
+        <li>FAQ</li>
+        <li>Certificates</li>
+        <li>Watch restoration </li>
+        
+      </ul>
+      
+      </div>
+      
+
+      <div class="footer--section">
+        <h2 class="footer--heading">Social</h2>
+        
+        <ul class="socials">
+          <li><i class="fab fa-2x fa-twitter"></i></li>
+          <li><i class="fab fa-2x fa-github"></i></li>
+          <li><i class="fab fa-2x fa-linkedin"></i></li>
+      
+        </ul>
+        
+        </div>
+    
```

Comment:
format the code in this file i.e., use proper indentation for readability. ex: use prettier

Reply:
ok I will surely do it next time